scripts/exregional_run_analysis.sh

```diff
@@ -1000,12 +1042,14 @@ fi
 #
 # comment out for testing
 
+if [ ${BKTYPE} -eq 0 ]; then
```

Comment:
@hu5970 Do you plan to turn off GSI at 03Z and 15Z spinup cycle for all the cases?

Reply:
The follow-up section do not dependent on the GSI execution, for example, GSI can run without any data.
But I do want to add machine dependent now because we only have trouble to run GSI cold start on WCOSS2.